content/docs/guides/ads_on_amp.md

```diff
@@ -0,0 +1,17 @@
+---
+$title: Ads on AMP
+$order: 8
+---
+<div class="toc">
+{% include "/views/partials/sub_nav.html" %}
+  <ul>
+    <li><a href="/learn/who/#ads">Supported Platforms</a></li>
+    <li><a href="/roadmap">Roadmap</a></li>
+  </ul>
+</div>
+
+AMP pages are just web pages, except consistently faster. Many of the best practices in this guide are consistent across non-AMP pages.
```

Comment:
Change to something like "AMP pages are just web pages with some custom web components and optimized delivery built-in." not true that they're just faster.